app/src/components/navigation/TopBar.vue

```diff
@@ -1,17 +1,21 @@
 <script setup lang="ts">
 import { PlayCircleIcon, ChevronLeftIcon } from "@heroicons/vue/24/solid";
 import ProfileMenu from "./ProfileMenu.vue";
-import { useRoute, RouterLink } from "vue-router";
+import { useRoute } from "vue-router";
 import DesktopMenu from "./DesktopMenu.vue";
 
 const route = useRoute();
+
+const goBack = () => {
+    window.history.back();
+};
 </script>
 
 <template>
     <header>
         <div class="z-40 bg-zinc-100 dark:bg-slate-800">
             <div class="flex flex-row items-center justify-between space-x-8 px-6 py-5">
-                <RouterLink to="/" class="flex items-center">
+                <div @click="goBack" class="flex cursor-pointer items-center">
```

Comment:
```suggestion
                <div @click="router.back()" class="flex cursor-pointer items-center">
```

Reply:
Implemented the change, much cleaner code, nice suggestion.